src/components/store/modals/ModalDappDetails.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <Modal>
+    <template v-slot:content>
+      <div class="tw-flex tw-flex-col tw-justify-center tw-items-center">
```

Comment:
The style won't change in dark mode
![Screen Shot 2021-10-03 at 1 36 25 PM](https://user-images.githubusercontent.com/40356749/135751987-9e210317-1a7e-4924-bf1c-076683387fc8.png)